src/cli/commands/config.js

```diff
@@ -19,8 +20,8 @@ export const {run, setFlags} = buildSubCommands('config', {
     if (args.length === 0 || args.length > 2) {
       return false;
     }
-
-    const [key, val = true] = args;
+    const key = args[0];
+    const val = BaseRegistry.normalizeConfigOption(args[1] || true);
```

Comment:
But maybe it's better to ensure config is normalized in yarn-registry's `saveHomeConfig` from the dependency prospective?